x-pack/test/functional/apps/transform/creation_index_pattern.ts

```diff
@@ -699,16 +712,21 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
           await transform.testExecution.logTestStep('should navigate to discover');
           await transform.table.clickTransformRowAction(testData.transformId, 'Discover');
-          await PageObjects.discover.waitUntilSearchingHasFinished();
+          await pageObjects.discover.waitUntilSearchingHasFinished();
 
           if (testData.discoverAdjustSuperDatePicker) {
+            await transform.testExecution.logTestStep(
+              `sets the date picker to the default '15 minutes ago'`
+            );
+            await transform.datePicker.openSuperDatePicker();
+            await transform.datePicker.quickSelect(15, 'm');
             await transform.discover.assertNoResults(testData.destinationIndex);
             await transform.testExecution.logTestStep(
               'should switch quick select lookback to years'
             );
-            await transform.discover.assertSuperDatePickerToggleQuickMenuButtonExists();
-            await transform.discover.openSuperDatePicker();
-            await transform.discover.quickSelectYears();
+            await transform.datePicker.assertSuperDatePickerToggleQuickMenuButtonExists();
```

Comment:
Nit: I know this is carried over from old code but I think this `assertSuperDatePickerToggleQuickMenuButtonExists` check should be included inside `openSuperDatePicker`, after which we won't need this check here.